src/components/PreferencesModal/TabButton.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { useCallback } from 'react'
-import styled from '../../lib/styled'
```

Comment:
I don't think this component is necessary either. 
just use SettingTabButton directly